common/src/main/java/gov/cms/ab2d/common/model/Job.java

```diff
@@ -69,6 +64,10 @@
     @Pattern(regexp = "ExplanationOfBenefits", message = "_type should be ExplanationOfBenefits")
     private String resourceTypes; // for now just limited to ExplanationOfBenefits
 
+    @OneToOne()
```

Comment:
Is this not a One-to-Many from Contract to Job? Technically, multiple `Job` records may refer to the same `Contract` as long as only one of those Jobs is in ACTIVE status.

Reply:
Yes it should be one to many from contract to job since those old job records would still be persisted.